cdap-common/src/main/java/io/cdap/cdap/common/http/CommonNettyHttpServiceBuilder.java

```diff
@@ -49,7 +49,7 @@ public void modify(ChannelPipeline pipeline) {
           EventExecutor executor = pipeline.context("dispatcher").executor();
           pipeline.addBefore(executor, "dispatcher", AUTHENTICATOR_NAME,
                              new AuthenticationChannelHandler(cConf.getBoolean(Constants.Security
-                                                                                 .ENFORCE_INTERNAL_AUTH)));
+                                                                                 .INTERNAL_AUTH_ENABLED)));
```

Comment:
This is in the `cdap-common` package and the `CommonNettyHttpServiceBuilder` seems to be used without injection in a lot of places, so switching it might be tricky and require changes in a lot of places. I can go ahead and perform this change in a subsequent refactor.